hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/SparkMain.java

```diff
@@ -49,17 +51,41 @@ public static void main(String[] args) throws Exception {
 
         JavaSparkContext jsc = SparkUtil.initJavaSparkConf("hoodie-cli-" + command);
         int returnCode = 0;
-        if (SparkCommand.ROLLBACK.equals(cmd)) {
-            assert (args.length == 3);
-            returnCode = rollback(jsc, args[1], args[2]);
-        } else if(SparkCommand.DEDUPLICATE.equals(cmd)) {
-            assert (args.length == 4);
-            returnCode = deduplicatePartitionPath(jsc, args[1], args[2], args[3]);
+        switch(cmd) {
+            case ROLLBACK:
+                assert (args.length == 3);
+                returnCode = rollback(jsc, args[1], args[2]);
+                break;
+            case DEDUPLICATE:
+                assert (args.length == 4);
+                returnCode = deduplicatePartitionPath(jsc, args[1], args[2], args[3]);
+                break;
+            case IMPORT:
+                assert (args.length == 11);
+                returnCode = dataImport(jsc, args[1], args[2], args[3], args[4], args[5], args[6],
+                        Integer.parseInt(args[7]), args[8], "yarn-client", args[9]);
```

Comment:
mainly because for all CLI commands it is currently hard-coded. SparkUtil#initJavaSparkConf(). Changing code to use that parameter instead so whenever we change that this too will get fixed.